src/Style/Book.module.css

```diff
@@ -0,0 +1,238 @@
+  width: 4rem;
+  height: 4rem;
+  background: transparent;
+  border-radius: 50%;
+}
+
+.progress {
+  display: flex;
+  align-items: center;
+  column-gap: 1vw;
+}
+
+.progressTest {
+  display: flex;
+  flex-direction: column;
+}
+
+.updateBtn {
+  width: 11rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: var(--azure);
+  color: white;
+}
+
+.bookInfo2 {
+  display: flex;
+  flex-direction: column;
+  row-gap: 1.43rem;
+}
+
+.chapter {
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.progcontainer {
+  display: flex;
+}
+
+pre {
+  background-color: white;
+}
+
+button {
+  background: none;
+  border: none;
+}
+
+.addBtn {
+  flex: auto;
+  font-size: medium;
+  width: 10rem;
+  height: 2.813rem;
+  margin: 1.1879999999999997rem 7.25rem 0 5rem;
+  padding: 0.801rem 2rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: var(--azure);
+  color: white;
+}
+
+.Formtitle {
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 6.25rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.formInput {
+  width: 100%;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 6.25rem;
+  padding: 0.813rem 15rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.formInput2 {
+  width: 80%;
+  height: 2.813rem;
+  margin: 1.1879999999999997rem 8.125rem 0;
+  padding: 0.375rem 0.938rem 0.563rem 2rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: #fff;
+}
+
+.formContainer {
+  display: flex;
+  justify-content: space-between;
+}
+
+.percent {
+  width: 4.25rem;
+  height: 2.438rem;
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  font-weight: normal;
+  color: var(--black-two);
+}
+
+.complete {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.wrec {
+  width: 5.625rem;
+  height: 5.62rem;
+  padding: 0.62rem 0.625rem 0.75rem 0.75rem;
+}
+
+.currentChapter {
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
```

Comment:
[OPTIONAL] Please follow the HTML&CSS Best Practices as per one of the requirements on this project. You have some repeating elements, you can combine them or give as a class attribute. You can check this article to learn more: https://speckyboy.com/good-bad-css-practices/